flink-runtime/src/test/java/org/apache/flink/runtime/rest/handler/legacy/utils/ArchivedExecutionConfigBuilder.java

```diff
@@ -75,6 +81,7 @@ public ArchivedExecutionConfigBuilder setPeriodicMaterializeIntervalMillis(
     public ArchivedExecutionConfig build() {
         return new ArchivedExecutionConfig(
                 executionMode != null ? executionMode : ExecutionMode.PIPELINED.name(),
+                jobType != null ? jobType : "UNKNOWN",
```

Comment:
If using the `UNKNOWN` as the default jobType here, why set `STREAMING` as the default value for `ConfigOption<String> JOB_TYPE`?